README.md

```diff
@@ -16,10 +18,16 @@ DokuWiki is a simple to use and highly versatile Open Source wiki software that
 
 ![](https://www.dokuwiki.org/_media/dokuwikimainwindow.png)
 
+## Demo
+
+* [YunoHost demo](https://demo.yunohost.org/dokuwiki/)
```

Comment:
Is there a link we can use on dokuwiki that would ask for connection ?

Reply:
I don't think so out of the box. Maybe thisshould be enought? https://demo.yunohost.org/dokuwiki/doku.php?id=start&do=login&sectok=

A plugin like this could be installed https://www.dokuwiki.org/plugin:authhttp


My researches

- https://www.dokuwiki.org/login
- https://forum.dokuwiki.org/thread/14638
- https://stackoverflow.com/questions/17447716/how-to-login-dokuwiki-from-my-websites

Reply:
Looks like it works with the link you gave me.
Just we should add login and pwd.

Reply:
Autologin works with this link : https://demo.yunohost.org/dokuwiki/doku.php?id=start&do=login&u=demo&p=demo

It's a bit ugly but well ...

Reply:
That's perfect !